assignment/src/com/assignmentweek4/springwithAnnotations/RandomFortuneService.java

```diff
@@ -0,0 +1,22 @@
+package com.assignmentweek4.springwithAnnotations;
+
+import java.util.Random;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class RandomFortuneService implements FortuneService {
+				private String[] data = { "Beware of Enemies",
+						"Be Kind",
+						"Be Brave",
+						};
+				private Random myRandom = new Random();
```

Comment:
Move this code inside getFortune() method as this logic is specific to that method.